samples/identity/fleet_provisioning.py

```diff
@@ -11,6 +11,33 @@
 from uuid import uuid4
 import json
 
+def link_command_line_utils():
+    """
+    An unfortunate necessary evil: We need to share the command_line_utils.py file across samples
+    but also need each sample in its own folder while still being directly executable. Python does
+    not like relative imports with non package/module python files, so we make a symlink if possible.
+    If it is not possible, we error out and direct where to find the file
```

Comment:
A sample should be something a user can copy/paste to start their own project.

Doing something so weird right off the bat ... doesn't seem like a great way to get someone started, who might not know IoT so well, who might not know Python so well, who might not know fancy OS stuff like soft links, etc.

Why not just structure these files in a way Python likes? Why does each sample need to be in its own directory? Like it was nice, but if it leads to shenanigans like this?

Here's an alternate way to structure things that Python would like:
```
samples/
    utils/
        command_line_utils.py
    README.md # contains links to each sample's docs 
    basic_connect.md
    basic_connect.py
    fleet_provisioning.md
    fleet_provisioning.py # contains: import utils.command_line_utils
    ... etc etc every sample in same directory
```